.php_cs.dist

```diff
@@ -0,0 +1,34 @@
+<?php
+
+$header = <<<'EOF'
```

Comment:
This header is managed by the php-formatter package. As you see, has been added in every file, and is already there :)

Reply:
So, should I remove the "header_comment" rule from PHP-CS-Fixer scope?

Reply:
Yes!
I will take a look during the next days if this package already covers this and can deprecate the php-formatter package. And then we'll change :)